_data/archiv/unternehmen_2022.json

```diff
@@ -5,7 +5,7 @@
         "caption": "",
         "logo": "/img/logos/AMPRION_LOGO_RGB_P.png",
         "url": "https://www.amprion.net/",
-        "ad": "img/catalogue_ads/A4_Warwel.jpg",
+        "ad": "/img/catalogue_ads/A4_Warwel.jpg",
```

Comment:
Is changing this to an absolute path right? I'm unsure. It worked before, why should we change it? (Only being pendantic here, because it happened to me in the  past, that links wich worked in development stopped working in production.)

Reply:
See my general comment